e2e/pages/events/manageDocumentsLAEvent.page.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
     c2DocumentsList: '#manageDocumentsSupportingC2List',
     supportingDocumentsForC2: supportingDocumentsFragment.supportingDocuments(0, 'temporaryC2Document_supportingEvidenceBundle'),
     supportingDocumentsCollectionId: '#supportingEvidenceDocumentsTemp',
-    supportingDocuments: function(index) {
+    supportingDocuments: function (index) {
```

Comment:
can be just:
supportingDocuments: index => supportingDocumentsFragment.supportingDocuments(index, 'supportingEvidenceDocumentsTemp')